server/utils/routesUtils/userRouteUtils.js

```diff
@@ -21,10 +21,10 @@ async function getProfileImage(id) {
       let image = 'data:image/(png|jpg);base64,' + encode(getImage.Body);
       return image;
     } else {
-      return 'uploads/images/profile-placeholder.gif';
+      return '/uploads/images/profile-placeholder.gif';
```

Comment:
the picture is missing on the server side public folder

Reply:
On your localhost you need to add it to the ``wikonnect/server/uploads/images`` here is the [absolute url](https://app.wikonnect.org/uploads/images/profile-placeholder.gif) to the image